src/main/java/service/DataParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package service;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitDataDto;
+
+public class DataParserImpl implements DataParser {
+
+    public static final String COMMA = ",";
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+    public static final int DATA_INDEX = 1;
+    public static final String WHITE_SPACE = " ";
+    public static final int FIRST_LINE_INDEX = 0;
+
+    @Override
+    public List<FruitDataDto> parseData(List<String> data) {
+        List<FruitDataDto> parsedData = new ArrayList<>();
+        DataValidator dataValidator = new DataValidatorImpl();
```

Comment:
make DataValidator class variable instead of local not to create its instance per each method call